### PR TITLE
1886/feat/ruler http fix

### PR DESCRIPTION
**Pull request by chenFriedman**

### CODE REVIEW

File: server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts [hunk 1]
```diff
@@ -1,16 +1,17 @@
 import { Router, Request, Response } from 'express';
+import axios from 'axios';
```
Comment (Noam6421) [inline server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts:2]: move up

File: server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts [hunk 2]
```diff
@@ -46,86 +47,16 @@ rulerRouter.post('/ruler/', UseCache, (request: Request, response: Response) =>
...
-        //     response.send(error);
-        // });
+    axios.post(rulerApiUrl, parameters).then((data) => {
+        rulerLogger.info(validAPIResponseLog, Severity.LOW);
+        setToCache(req.originalUrl, data.data);
+        res.send(data.data);
+    }
+    ).catch((err: any) => {
+        rulerLogger.error(invalidAPIResponseLog(err), Severity.HIGH);
+        res.send(err);
```
Comment (Noam6421) [inline server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts:58]: add .status(errorStatusCode).send(error);

**Review by Noam6421 — CHANGES_REQUESTED** [review]

File: server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts (hunk 2 above)
Comment (chenFriedman) [reply server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts:58]: errorStatusCode is part of GraphqlHTTPRequest
I'll look for a way to get it here

Comment (chenFriedman) [reply server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts:58]: oops it's just means 500

Comment (Noam6421) [reply server/src/ClientToDBAPI/RulerRoute/mainRoutes.ts]: haha yes I was also surprised from this

**Review by Noam6421 — APPROVED** [review]

**Pull request merged**